crates/indexer/src/models/system_transactions.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::transactions::{Transaction, TransactionQuery};
+use crate::{schema::system_transactions, util::parse_timestamp};
+use aptos_api_types::SystemTransaction as APISystemTransaction;
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Associations, Clone, Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize,
+)]
+#[diesel(belongs_to(Transaction, foreign_key = version))]
+#[diesel(primary_key(version))]
+#[diesel(table_name = system_transactions)]
+pub struct SystemTransaction {
+    pub version: i64,
+    pub block_height: i64,
+    pub txn_serialized: Vec<u8>,
```

Comment:
Do we really need to store the raw txn?

Reply:
@bowenyang007

Reply:
All the indexer changes in this PR so far were made just to get rid of compile errors...
It doesn't have to be like this :)

Reply:
Yea I can clean everything in this crate up. This is all unnecessary b/c we're no longer adding new features to this part of the indexer.